plugin/docker/container.go

```diff
@@ -51,7 +52,8 @@ func (c *container) List(ctx context.Context) ([]plugin.Entry, error) {
 	clfAttr.SetSize(uint64(content.Size()))
 	clf.SetAttributes(clfAttr)
 
-	return []plugin.Entry{cm, clf}, nil
+	// Include a view of the remote filesystem using volume.FS
+	return []plugin.Entry{cm, clf, vol.NewFS("fs", c)}, nil
```

Comment:
Hmm, we may need a config setting for this to translate to external plugins. Or maybe external plugins don't get to have this benefit. I guess the latter's the likely case here, given that introducing this for external plugins would cause a dependency cycle.

Reply:
I still think external plugins should be a separate package, which would solve the dependency cycle.

Reply:
Yes, I think we could place them in a `plugin/internal` package so they're not exported in godocs. May need to fix some things with the Registry when registering external plugins in order to avoid a `plugin => internal => plugin` dep. cycle

Reply:
Yes, tweaks to the Registry would be needed.